README.md

```diff
@@ -19,33 +19,68 @@ Composer will automatically install dependencies, register standards paths, and
 To include standards as part of a project require them as development dependencies:
 
 ```bash
-composer require yoast/yoastcs:dev-master --dev
+composer require --dev yoast/yoastcs:^1.0
 ```
 
-Note that Composer won't run configuration scripts in this scenario and the root project needs to take care of it.
+Composer will automatically install dependencies and register the YoastCS and other external standards with PHP_CodeSniffer.
 
 ## PHP Code Sniffer
 
 Set of [PHP Code Sniffer](https://github.com/squizlabs/PHP_CodeSniffer) rules.
 
-Based on [WordPress Coding Standards](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards) project, implementing official [WordPress PHP Coding Standards](https://make.wordpress.org/core/handbook/coding-standards/php/).
-
 Severity levels:
 
  - error level issues are considered mandatory to fix in Yoast projects and enforced in continuous integration
  - warning level issues are considered recommended to fix
 
-### Command line
+### The YoastCS Standard
+
+The `Yoast` standard for PHP_CodeSniffer is comprised of the following:
+* The `WordPress` ruleset from the [WordPress Coding Standards](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards) implementing the official [WordPress PHP Coding Standards](https://make.wordpress.org/core/handbook/coding-standards/php/), with some select exclusions.
+* The [`PHPCompatibilityWP`](https://github.com/PHPCompatibility/PHPCompatibilityWP) ruleset which checks code for PHP cross-version compatibility while preventing false positives for functionality polyfilled within WordPress.
+* A number of custom Yoast specific sniffs.
+* A number of additional sniffs taken from `PHP_CodeSniffer`.
+
+Files within version management and dependency related directories, such as for instance the Composer `vendor` directory, are not scanned by default.
+
+#### Sniffs
+
+To obtain a list of all sniffs used within YoastCS:
+```bash
+"vendor/bin/phpcs" -e --standard=Yoast
+```
+
+#### Sniff Documentation
+
+Not all sniffs have documentation available about what they sniff for, but for those which do, this documentation can be viewed from the command-line:
+```bash
+"vendor/bin/phpcs" --standard=Yoast --generator=text
+```
+
+### Running the sniffs
+
+#### Command line
 
 ```bash
 "vendor/bin/phpcs" --extensions=php /path/to/folder/
 ```
 
-### PhpStorm
+For more command-line options, please have a read through the [PHP_CodeSniffer documentation](https://github.com/squizlabs/PHP_CodeSniffer/wiki/Usage).
+
+#### Yoast plugin repositories
+
+All Yoast plugin repositories contain a `[.]phpcs.xml.dist` file contain the repository specific configuration.
+
+From the root of these repositories, you can run PHPCS by using:
+```bash
+composer check-cs
+```
+
+#### PhpStorm
 
 Refer to [Using PHP Code Sniffer Tool](https://www.jetbrains.com/phpstorm/help/using-php-code-sniffer-tool.html) in PhpStorm documentation.
```

Comment:
```suggestion
Refer to [Using PHP Code Sniffer Tool](https://www.jetbrains.com/phpstorm/help/using-php-code-sniffer-tool.html) in the PhpStorm documentation.
```

Reply:
Fair point, but wasn't part of this PR ;-)